lunes_cms/README.md

```diff
@@ -19,7 +19,7 @@ This is the content management system for the vocabulary trainer app Lunes, whic
 
 Following packages are required before installing the project (install them with your package manager):
 
-* python3.8 or greater
+* python3.13 or greater
```

Comment:
We have Python 3.11 on our servers. We should support that version as well.

Reply:
Also Django 5.0 officially only supports Python up to 3.12.

Reply:
Thanks Sven! I just saw the `Python >=3.10` specification on the PyPI page. I've changed to Python `3.11`.